recipes/at-spi2-core/meta.yaml

```diff
@@ -0,0 +1,64 @@
+# Prior to conda-forge, Copyright 2014-2018 Peter Williams and collaborators.
+# This file is licensed under a 3-clause BSD license; see LICENSE.txt.
+
+{% set name = "at-spi2-core" %}
+{% set version = "2.35.1" %}
+{% set version_majmin = '.'.join(version.split('.', 2)[:2]) %}
+{% set sha256 = "ad97c7b6071c21ed59aeb092f3bdc592cf9edbca2710ff428d7829d65675a185" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://ftp.gnome.org/pub/gnome/sources/{{ name }}/{{ version_majmin }}/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    - {{ pin_subpackage('at-spi2-core', max_pin='x.x') }}
+  missing_dso_whitelist:  # [linux]
+    - '*/libX11.so*'  # [linux]
```

Comment:
Is the above needed if we are using CDTs?

Reply:
The issue is that because `xorg-x11` is installed through the `cairo` dependency, the DSO checking picks up that the non-CDT library is linked in but that it is not present as a run requirement, so the check without fail without something like this.

Reply:
Ah, okay